docs/source/pages/documentation.md

```diff
@@ -1,18 +1,170 @@
 # Full Documentation
 
-Datashuttle is a work in progress as has not been officially released. It is not ready for use
-as documented, please await first official release.
 
-DataShuttle helps to manage and transfer a project with many "local" machines all connected to a "central" machine.
-DataShuttle has functions to help:
-* Generate BIDS-formatted folder structures
-* Transfer data between central and local machines
+DataShuttle is a tool to streamline the management of neuroscience projects, allowing simple and convenient standardisation of data structures.
```

Comment:
Do we mean "data structures" here? I'm thinking more "files and folders"?

Reply:
Also, should the fact that it's a Python and Command Line tool already be mentioned here? Will be useful for potential users to decide early whether `datashuttle` is what they want or not (I know it's mentioned further down)

Reply:
Yes I think this is a good idea. I'm in a muddle as to where to introduce this, because as you say it is important to know sooner rather than later. However the terms API and CLI are a little technical and might not be immediately obvious even for those from a non-computer science background who know how to code or use a terminal (e.g. I recall not knowing what an API was even though I was reasonably experienced with python).  As such I don't want to introduce it too early and make it seem overly techy.